src/site/content/en/metrics/inp/index.md

```diff
@@ -158,56 +157,22 @@ INP can be measured both in [the field](/lab-and-field-data-differences/#field-d
 The best way to measure your website's INP is by gathering metrics from actual users in the field. If you're accustomed to relying on lab data for assessing performance, take some time to read [Why lab and field data can be different (and what to do about it)](/lab-and-field-data-differences/).
 {% endAside %}
 
-### Field tools
-
-- [PageSpeed Insights](https://pagespeed.web.dev).
-- [Chrome User Experience Report (CrUX)](https://developer.chrome.com/docs/crux/).
-  - Via BigQuery in the CrUX dataset's `interaction_to_next_paint` table.
-  - CrUX API via `interaction_to_next_paint`.
-  - CrUX Dashboard.
-- [`web-vitals` JavaScript library](https://github.com/GoogleChrome/web-vitals).
-
-### Lab tools
+### In the field
 
-- Lighthouse Panel in DevTools, available in "Timespan Mode".
-- [Lighthouse npm module](https://www.npmjs.com/package/lighthouse).
-- [Lighthouse User Flows](/lighthouse-user-flows/).
-- [Web Vitals extension for Chrome](https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma).
+Ideally, your journey in optimizing INP will start with field data. At its best, field data from a Real User Monitoring (RUM) provider will give you not only a page's INP value, but also contextual data that highlights what specific interaction was responsible for the INP value itself, whether the interaction occurred during or after page load, the type of interaction (click, keypress, or tap), and other valuable information.
```

Comment:
IMHO web-vitals is one of the best for this (and the pioneer!) so let's drop the word "provider":

```suggestion
Ideally, your journey in optimizing INP will start with field data. At its best, field data from Real User Monitoring (RUM) will give you not only a page's INP value, but also contextual data that highlights what specific interaction was responsible for the INP value itself, whether the interaction occurred during or after page load, the type of interaction (click, keypress, or tap), and other valuable information.
```